### PR TITLE
[NB-5478] Introduced Notebook GPU with Rapids Built-in Custom Environment

### PR DESCRIPTION
**Pull request by lyapun**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary

Introduced Notebook GPU with Rapids Built-in Custom Environment

## Rationale

https://datarobot.atlassian.net/browse/NB-5478

### CODE REVIEW

File: public_dropin_notebook_environments/python39_notebook_gpu_rapids/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+## Supported Libraries
+
+This environment has been built for python 3.9 and includes commonly used OSS machine learning and data science libraries.
+For specific version information, see [requirements](requirements.txt).
+
+## Instructions
+
+1. Update [requirements](requirements.txt) to add your custom libraries supported by Python 3.9.
+2. From the terminal, run `tar -czvf py39_notebook_dropin.tar.gz -C /path/to/public_dropin_notebook_environments/python39_notebook/ .`
```
Comment (c-h-russell-walker) [inline public_dropin_notebook_environments/python39_notebook_gpu_rapids/README.md:13]: Kind of a nit - should this be: ??

```suggestion
2. From the terminal, run `tar -czvf py39_notebook_dropin.tar.gz -C /path/to/public_dropin_notebook_environments/python39_notebook_gpu_rapids/ .`
```

File: public_dropin_notebook_environments/python39_notebook_gpu_rapids/agent/agent.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Copyright 2022 DataRobot, Inc. and its affiliates.
```
Comment (c-h-russell-walker) [inline public_dropin_notebook_environments/python39_notebook_gpu_rapids/agent/agent.py:1]: Sorry, again a very small nit but do we want this to be: ??

```suggestion
# Copyright 2024 DataRobot, Inc. and its affiliates.
```

**Review by c-h-russell-walker — APPROVED** [review]
Looks good 👍 

I spotted what I think could be an update to make in a README as well as maybe a bunch of the files have a different year in their copyright - uncertain if you want/need to update those.

**Comment by lyapun** [comment]
Thanks, @c-h-russell-walker! Applied your suggestions

**Review by yakov-g — APPROVED** [review]

**Pull request merged**